apps/api/src/app/user/usecases/update-name-and-profile-picture/update-name-and-profile-picture.command.ts

```diff
@@ -0,0 +1,16 @@
+import { IsDefined, IsOptional, IsString, IsUrl } from 'class-validator';
+import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
+
+export class UpdateNameAndProfilePictureCommand extends EnvironmentWithUserCommand {
+  @IsUrl({ require_tld: false })
+  @IsOptional()
+  imageUrl: string;
+
+  @IsDefined()
+  @IsString()
+  firstName: string;
+
+  @IsString()
+  @IsDefined()
+  lastName: string;
```

Comment:
used firstName and lastName as we use similar properties for signup and also in user entity